.github/workflows/cmake.yml

```diff
@@ -18,34 +18,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v3
-      with:
-        repository: villevoutilainen/wg21_p2300_std_execution
-        ref: member-only-customization-points
-        path: stdexec
-
-    - name: Configure stdexec
-      run: cmake -B ${{github.workspace}}/stdexec-build -S ${{github.workspace}}/stdexec -DCMAKE_INSTALL_PREFIX=${{github.workspace}}/stdexec-install
-
-    - name: Install stdexec
-      run: cmake --install ${{github.workspace}}/stdexec-build
-
-    - uses: actions/checkout@v3
-      with:
-        repository: catchorg/Catch2
-        path: catch2
-        ref: v2.13.6
-
-    - name: Configure Catch2
-      run: cmake -B ${{github.workspace}}/catch2-build -S ${{github.workspace}}/catch2 -DCMAKE_INSTALL_PREFIX=${{github.workspace}}/catch2-install -DBUILD_TESTING=OFF
-
-    - name: Build Catch2
-      run: cmake --build ${{github.workspace}}/catch2-build --parallel 4
-
-    - name: Install Catch2
-      run: cmake --install ${{github.workspace}}/catch2-build
-
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
```

Comment:
awesome

Reply:
we don't need to pass stdexec_DIR and catch2_dir to cmake if we use fetchcontent, right?

Reply:
Yes, we don't need to pass stdexec_DIR and catch2_DIR to cmake since "fetchcontent"  can automatically download these libraries in first time compilation.